src/components/Tool/Appeal/Shared/AmountAndFrequency/AmountAndFrequency.tsx

```diff
@@ -0,0 +1,25 @@
+import { UseGetPledgeOrDonation } from 'src/hooks/useGetPledgeOrDonation';
+import theme from 'src/theme';
+
+export const AmountAndFrequency: React.FC<
+  Omit<UseGetPledgeOrDonation, 'pledgeValues' | 'pledgeDonations'>
```

Comment:
Instead of omitting the props you aren't using, it could be clearer to pick the props you do use.

```suggestion
  Pick<UseGetPledgeOrDonation, 'amountAndFrequency' | 'pledgeOverdue'>
```